portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -385,6 +385,47 @@ definition {
 			searchTerm = "");
 	}
 
+	@description = "This is a use case for LPS-179990"
+	@priority = 3
+	test CanRemoveSortFieldsAndDeactivateRelevanceInSortWidget {
+		task ("Open search page and add a new Sort Widget") {
+			Search.openSearchPage();
+
+			SearchPortlets.addWidgetsViaUI(searchPortletList = "Sort");
+
+			Refresh();
+		}
+
+		task ("Open Sort Widget configuration") {
+			Portlet.gotoPortletOptions(
+				portletName = "Sort",
+				portletOption = "Configuration");
+
+			SelectFrame(locator1 = "IFrame#CONFIGURATION");
+		}
```

Comment:
Avoid creating task blocks for navigation. We can assume the reader knows how to navigate to locate things. This block can be grouped with the next task block.